src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -429,6 +434,69 @@ def generate_request_payload(configuration, location, public_key, tags):
     return cc
 
 
+def get_kubeconfig_node_dict(kube_config=None):
+    if kube_config is None:
+        kube_config = os.getenv('KUBECONFIG') if os.getenv('KUBECONFIG') else os.path.join(os.path.expanduser('~'), '.kube', 'config')
+    try:
+        kubeconfig_data = config.kube_config._get_kube_config_loader_for_yaml_file(kube_config)._config
+    except Exception as ex:
+        telemetry.set_exception(exception=ex, fault_type=consts.Load_Kubeconfig_Fault_Type,
```

Comment:
This should be set as user fault if an exception has occured

Reply:
Okay, will do.